### PR TITLE
fixed opening directories on linux

### PR DESCRIPTION
**Pull request by shocktail39**
the first / of the file path was getting removed, causing interface to try to access a relative path instead of an absolute one.

### CODE REVIEW

File: interface/src/Application.cpp [hunk 1]
```diff
@@ -4168,7 +4168,7 @@ void Application::loadErrorDomain(QUrl domainURL) {
 bool Application::importImage(const QString& urlString) {
     qCDebug(interfaceapp) << "An image file has been dropped in";
     QString filepath(urlString);
-    filepath.remove("file:///");
+    filepath.remove("file://");
```
Comment (daleglass) [inline interface/src/Application.cpp:4171]: Technically, this shouldn't be here because it's a different problem being fixed

**Review by daleglass — CHANGES_REQUESTED** [review]

File: interface/src/Application.cpp (hunk 1 above)
Comment (shocktail39) [reply interface/src/Application.cpp:4171]: ah, should i remove it and put it in its own request?

Comment (daleglass) [reply interface/src/Application.cpp:4171]: That, or you could change the description of the commit to make it clearer the change that was made

Comment (shocktail39) [reply interface/src/Application.cpp:4171]: ok will do

**Comment by shocktail39** [comment]
needs testing on windows to make sure fixing it on linux didn't break it on windows.

**Review by daleglass — APPROVED** [review]

**Comment by AnimatedMannequin** [comment]
Here are some builds you may use for testing: 

**ubuntu-18.04, full**
 - https://public.overte.org/build/overte/PR142/Overte-PR142-65a5f29.tar.Z
 - https://public.overte.org/build/overte/PR142/Overte-PR142-65a5f29.sh
 - https://public.overte.org/build/overte/PR142/Overte-PR142-65a5f29.tar.gz

**macOS-10.15, full**
 - https://public.overte.org/build/overte/PR142/Overte-PR142-65a5f29.dmg

**windows-2019, full**
 - https://public.overte.org/build/overte/PR142/Overte-PR142-65a5f29.exe

**self-hosted_debian-11_aarch64, full**
 - https://public.overte.org/build/overte/PR142/Overte-PR142-65a5f29.tar.Z
 - https://public.overte.org/build/overte/PR142/Overte-PR142-65a5f29.tar.gz
 - https://public.overte.org/build/overte/PR142/Overte-PR142-65a5f29.sh

**Comment by ksuprynowicz** [comment]
It breaks uploading images by drag and drop on Windows.

**Comment by ksuprynowicz** [comment]
All I get in the logs is:
```
[07/30 15:06:19] [DEBUG] [hifi.interface] An image file has been dropped in
[07/30 15:06:19] [DEBUG] [hifi.ui] setKeyboardRaised:  QQuickFlow(0x236aa55bc60) , raised:  false , numeric:  false , password:  false
[07/30 15:06:19] [WARNING] [hifi.interface] "Error downloading model: Could not upload model to the Asset Server."```

**Comment by shocktail39** [comment]
i'm not around a computer at the moment.  could someone please try reverting line 4171 back to `filepath.remove("file:///");` and see if that works on both?  i just realized that i initially determined it was "broken" in serverless, where there is no asset server to upload to.

**Comment by ksuprynowicz** [comment]
Sure! I willo give it a try

**Comment by AnimatedMannequin** [comment]
Here are some builds you may use for testing: 

**ubuntu-18.04, full**
 - https://public.overte.org/build/overte/PR142/Overte-PR142-3abfb23.tar.Z
 - https://public.overte.org/build/overte/PR142/Overte-PR142-3abfb23.tar.gz
 - https://public.overte.org/build/overte/PR142/Overte-PR142-3abfb23.sh

**macOS-10.15, full**
 - https://public.overte.org/build/overte/PR142/Overte-PR142-3abfb23.dmg

**self-hosted_debian-11_aarch64, full**
 - https://public.overte.org/build/overte/PR142/Overte-PR142-3abfb23.tar.Z
 - https://public.overte.org/build/overte/PR142/Overte-PR142-3abfb23.sh
 - https://public.overte.org/build/overte/PR142/Overte-PR142-3abfb23.tar.gz

**windows-2019, full**
 - https://public.overte.org/build/overte/PR142/Overte-PR142-3abfb23.exe

**Comment by JulianGro** [comment]
Fixed in https://github.com/overte-org/overte/pull/158
Thanks for finding the exact place where the problem occurs.

**Pull request closed without merging**